tests/sqlparser_mssql.rs

```diff
@@ -1393,6 +1393,52 @@ fn parse_mssql_declare() {
     let _ = ms().verified_stmt(declare_cursor_for_select);
 }
 
+#[test]
+fn test_mssql_cursor() {
+    let full_cursor_usage = "\
+        DECLARE Employee_Cursor CURSOR FOR \
+        SELECT LastName, FirstName \
+        FROM AdventureWorks2022.HumanResources.vEmployee \
+        WHERE LastName LIKE 'B%'; \
+        \
+        OPEN Employee_Cursor; \
+        \
+        FETCH NEXT FROM Employee_Cursor; \
+        \
+        WHILE @@FETCH_STATUS = 0 \
+        BEGIN \
+            FETCH NEXT FROM Employee_Cursor; \
+        END; \
+        \
+        CLOSE Employee_Cursor; \
+        DEALLOCATE Employee_Cursor\
+    ";
+    let _ = ms().statements_parse_to(full_cursor_usage, 6, "");
+}
+
+#[test]
+fn test_mssql_while_statement() {
+    let while_single_statement = "WHILE 1 = 0 PRINT 'Hello World';";
```

Comment:
Can we include a test case with multiple statements in the while block?

Also since this introduces a new statement, can we include a test case that asserts the returned AST?

Reply:
> Can we include a test case with multiple statements in the while block?

This is covered by the additional subsequent examples

> Also since this introduces a new statement, can we include a test case that asserts the returned AST?

Yes, I'll do that here for the initial example case

Reply:
Done 👍